packages/grafana-eslint-rules/rules/no-aria-label-e2e-selectors.cjs

```diff
@@ -0,0 +1,99 @@
+const GRAFANA_E2E_PACKAGE_NAME = '@grafana/e2e-selectors';
+
+// A relative simple lint rule that will look of the `selectors` export from @grafana/e2e-selectors
+// is used in an aria-label
+// There's probably a few ways around this, but the point isn't to be exhaustive but to find the
+// majority of instances to count them
+function noAriaLabelE2eSelectors(context) {
```

Comment:
I wish this was typescript (the AST is pretty exhaustively typed which helps you out a lot!), but eslint (understandably) can only load JS plugins and it was too much futzing around to add in a build pipeline to to the TS -> JS.

I'm not even sure the best way to do this - don't want to convert this on every lint (it changes rarely), maybe spin it out into a separate repo and publish to NPM? 🤷🏻 

If we ever add a second rule, that's when I would want to move this to typescript.

Reply:
Did you check https://typescript-eslint.io/custom-rules/ ?

Reply:
Yeah seen that, but disregarded because we're still writing javascript. Would definitely use if we were actually writing Typescript here.

I briefly tried to use js/tsdoc comments to get typescript to typecheck javascript, but it was just too fiddly that I gave up on it. I might see if it's easier with using those utils though.